assignment3.txt

```diff
@@ -0,0 +1,6 @@
+Jack Rosen
+1. This is valid but it is assumed that the variable i has already been initialized.
+2. This is an infinite loop since i is always below 10 because it only gets smaller than 0 and never reaches 10.
```

Comment:
This is true, but you missed the fact that there's also a semicolon after the for loop!